code/game/sound.dm

```diff
@@ -55,7 +55,8 @@ var/global/const/FALLOFF_SOUNDS = 0.5
 
 	volume *= pressure_factor
 
-	if(istype(T,/turf/simulated) && istype(turf_source,/turf/simulated))
+	//Dense turfs are not in the zones, so they shouldn't be penalized here when source
+	if(!turf_source.density && istype(T,/turf/simulated) && istype(turf_source,/turf/simulated))
```

Comment:
```suggestion
	if(!turf_source.blocks_air && istype(T,/turf/simulated) && istype(turf_source,/turf/simulated))
```
I'm not 100% sure about this, but it seems better to penalize it based off of whether or not it can participate in a zone (i.e. if it blocks air or not), rather than just using density as a proxy. Haven't tested, could be wrong.